src/main/java/seedu/address/model/claim/ClaimSet.java

```diff
@@ -0,0 +1,208 @@
+package seedu.address.model.claim;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * A class to store a set of {@code Claim}.
+ * This class ensures that a single {@code ClaimSet} cannot have multiple claims with the same {@code Claim}
+ * description.
+ */
+public class ClaimSet implements Set<Claim> {
```

Comment:
actually i dont think it should be set, because how are we going to edit specific claims

Reply:
hmm, may I ask why? I feel like using a set makes use defensive programming to ensure there can never be duplicate claims. Is there anything that a list can do better in?

Reply:
i think we would require indexing for the edit claims logic, cuz we need to get or set a claim at that index